client/src/components/ProductDetailPage.jsx

```diff
@@ -44,6 +49,9 @@ class ProductDetailPage extends React.Component {
         <RelatedItemsAndOutfit />
         <QuestionsAndAnswers />
         <RatingsAndReviews onChangeFileHandler={this.onChangeFileHandler} onClickUploadHandler={this.onClickUploadHandler} />
+        <div>{console.log(`Test of mock data passed to render function:\n Mock Products: ${productsMock} \n
```

Comment:
Should probably remove this console log now, if we know the mock data works

Reply:
Will do. I'll resubmit after removing.